### PR TITLE
[monitoring-applications] fixes rabbitmq dashboard using job rabbitmq

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
fixes rabbitmq dashboard using job rabbitmq

## Why do we need it, and what problem does it solve?
fixes dashboard when using custom label

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: monitoring-applications
type: chore
summary: Fixes rabbitmq dashboard using job rabbitmq.
impact_level: low
```

### CODE REVIEW

**Comment by alexey-gavrilov-flant** [comment]
cherry-pick  to stable channel

**Comment by alexey-gavrilov-flant** [comment]
https://flant.slack.com/archives/CML93AWG2/p1663687033024089?thread_ts=1663683757.925339&cid=CML93AWG2

**Comment by nabokihms** [comment]
@alexey-gavrilov-flant, I didn't get an idea how this PR fixes the dashboard. Could you please elaborate more?

**Review by nabokihms — CHANGES_REQUESTED** [review]
Let's talk more about the case because, as I see it, this PR limits all queries to some custom jobs.

**Comment by alexey-gavrilov-flant** [comment]
I set up a queue length alert, but then the dashboard display breaks. because there are no filters for custom services.
```
---
apiVersion: v1
kind: Service
metadata:
  labels:
    prometheus.deckhouse.io/custom-target: rabbitmq-queue
  annotations:
    prometheus.deckhouse.io/path: /metrics/detailed
    prometheus.deckhouse.io/query-param-family: queue_coarse_metrics
  name: rabbitmq-queue
spec:
  ports:
  - name: http-metrics
    port: 15692
    protocol: TCP
    targetPort: 15692
  selector:
    app.kubernetes.io/name: rabbitmq
  type: ClusterIP
```

**Comment by alexey-gavrilov-flant** [comment]
these two queries give part of the same metrics, which is the reason for not working.

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json [hunk 1]
```diff
@@ -159,7 +159,7 @@
       "pluginVersion": "7.0.6",
       "targets": [
         {
-          "expr": "sum(rate(rabbitmq_channel_messages_published_total[60s]) * on(instance) group_left(namespace) rabbitmq_identity_info{namespace=\"$namespace\"})",
+          "expr": "sum(rate(rabbitmq_channel_messages_published_total[60s]) * on(instance) group_left(namespace) rabbitmq_identity_info{namespace=\"$namespace\",job !~ \"^custom-.*\"})",
```
Comment (nabokihms) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json:162]:
```suggestion
          "expr": "sum(rate(rabbitmq_channel_messages_published_total[60s]) * on(instance) group_left(namespace) rabbitmq_identity_info{namespace=\"$namespace\",job=\"rabbitmq\"})",
```

**Review by nabokihms — COMMENTED** [review]
Instead of excluding, let's include only rabbitmq jobs

**Pull request merged**